source/octf/utils/table/Table.h

```diff
@@ -219,7 +151,8 @@ Row &operator<<(Row &row, const ::google::protobuf::Descriptor *desc);
  * bar.set_name("Star");
  *
  * // Fill table's header (row 0)
- * table[0] << foo.GetDescriptor() << bar.GetDescriptor();
+ * table::setHeader(table[0], foo);
+ * table::setHeader(table[0], bar);
```

Comment:
Is the intended use to call setHeader with each added (streamed) message?

Reply:
Those two messages are streamed into table, so we need to build a header using them both.